### PR TITLE
Desertbeachtileresort

### PR DESCRIPTION
**Pull request by GSonderling**
I reorganized the templates for desert beaches, as requested in #13587 .

The excess commits are result of unfortunate accident with notepad++.

### CODE REVIEW

**Comment by abcdefg30** [comment]
Can you squash the commits together, please?
(See https://help.github.com/articles/about-git-rebase/ and http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html , you can also join [#openra on IRC](http://webchat.freenode.net/?channels=openra) if you need further assistance.)

**Comment by GSonderling** [comment]
There is a slight issue with that. You see unfortunately pushed my commits without author.

Since rebase requires identity for all commits it blocks my attempts to squash commits despite identities now being set (in both global and local scope, I double checked). 

I can retract this pull request, delete the branch and instead create new branch with just one nice commit. But I can't really alter history of this one.

**Comment by abcdefg30** [comment]
That sounds weird, `git rebase` shouldn't require any identification. What is the error message you get?

**Comment by GSonderling** [comment]
`fatal: empty ident name (for <mydomainID>) not allowed`

I aborted the rebase and reentered all relevant info (user.name and user.email), same error. 
Looked around and only new relevant information I found was [](https://github.com/Microsoft/BashOnWindows/issues/2203), which did seem to fit my issue.

I admit that it is weird, I would expect that if that kind of thing mattered it would have shown up earlier. 

Anyway seeing that I lack any other explanation I went with one provided above. Although I am open to alternatives.

**Comment by GraionDilach** [comment]
A mixed reset (leave working directory unlisted, reset index)to the last bleed commit and then recommitting your changes with a force-push thenafter sounds like the solution, but notsure how to call that in the CLT.

**Comment by abcdefg30** [comment]
`git reset --soft bleed` and then force push. Yeah, that should work.

**Comment by GSonderling** [comment]
I cleaned up the commits, thanks for the help.

File: mods/ra/tilesets/desert.yaml [hunk 1]
```diff
@@ -2316,6 +2316,52 @@ Templates:
...
+	Template@440:
+		Id: 440
+		Images: sh63.des
+		Size: 4,1
+		Categories: Beach
+		Tiles:
+			0: Beach
+			1: Beach
+			2: Beach
+			3: Beach
```
Comment (reaperrr) [inline mods/ra/tilesets/desert.yaml]: Some spurious spaces after 'Beach' here, please remove them.

File: mods/ra/tilesets/desert.yaml [hunk 2]
```diff
@@ -2540,7 +2549,7 @@ Templates:
 			4: Clear
 			6: Water
 			7: Beach
-			8: Beach
+			8: Beach
```
Comment (reaperrr) [inline mods/ra/tilesets/desert.yaml]: Some spurious tabs after 'Beach' here, please remove them.

**Comment by GSonderling** [comment]
Deleted spaces and tabs as requested.

File: mods/ra/tilesets/desert.yaml (hunk 2 above)
Comment (GSonderling) [inline mods/ra/tilesets/desert.yaml]: Tabs deleted.

File: mods/ra/tilesets/desert.yaml (hunk 1 above)
Comment (GSonderling) [inline mods/ra/tilesets/desert.yaml]: Spaces deleted.

**Review by abcdefg30 — APPROVED** [review]
Can you please rebase and squash the commits again after those trailing spaces are removed? ^^
Looks good to me otherwise. 👍

**Comment by GSonderling** [comment]
Commits squashed.

File: mods/ra/tilesets/desert.yaml [hunk 3]
```diff
@@ -2718,28 +2676,70 @@ Templates:
...
+		Images: sh64.des
+		Size: 2,3
+		Categories: Beach
+		Tiles:
+			0: River
+			1: Rough
+			2: River
+			3: River
+			4: River
+			5: Rock
```
Comment (ltem) [inline mods/ra/tilesets/desert.yaml]: One more trailing space found. You can use `git commit --amend`  to attach the change to the last commit, so you don't need create a new commit and squash it again, after that I would approve this PR.

**Review by ltem — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/63a60e07d6641aaef4d722282988a3cb56e9231b...791b3c02a17f9048dea2977d36546c93ac48545a)

Thanks. :)

**Pull request merged**